pathdb/datastax/src/main/java/org/commonjava/storage/pathmapped/pathdb/datastax/CassandraPathDB.java

```diff
@@ -542,18 +574,12 @@ public boolean copy( String fromFileSystem, String fromPath, String toFileSystem
             delete( toFileSystem, toPath );
         }
 
-        // check parent paths
-        String fromParentPath = PathMapUtils.getParentPath( fromPath );
         String toParentPath = PathMapUtils.getParentPath( toPath );
-        if ( fromParentPath != null && !fromParentPath.equals( toParentPath ) )
-        {
-            makeDirs( toFileSystem, toParentPath );
-        }
-
```

Comment:
Little confused here. What's the purpose to remove this mkdir operation?

Reply:
The logic is moved to insert() method. During insert(), the missing dirs will be created.